### PR TITLE
Add an extractMsg option to convert msg into message.

### PR DESCRIPTION
**Pull request by jum**
The short form msg is not recognized by stackdriver logging but used in many go programs, for example any that use log/slog.

### CODE REVIEW

**Comment by nanoandrew4** [comment]
Please document the new log option with a description in the README

File: README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ The following [log-opts](https://docs.docker.com/config/containers/logging/confi
 | extract-json-message | true    | Enables unmarshalling JSON messages and sending the jsonPayload as the unmarshalled map. Kind of the whole point of this plugin, but you can disable it so it behaves just like the `gcplogs` plugin if you wish                                                          |
 | local-logging        | false   | Enables logging to a local file, so logs can be viewed with the `docker logs` command. If false, the command will show no output                                                                                                                                          |
 | extract-severity     | true    | Extracts the severity from JSON logs to set them for the log that will be sent to GCP. It will be removed from the jsonPayload section, since it is set at the root level. Currently the supported severity field names to extract are the following: `severity`, `level` |
+| extract-msg     | true    | Extracts the msg field from JSON logs to set the message field GCP expects. It will be removed from the jsonPayload section, since it is set at the root level. Fields named msg are producted for example by the golang log/slog package. |
```
Comment (nanoandrew4) [inline README.md:133]: Typo, should be `produced` not `producted`

**Comment by jum** [comment]
Ouch, I merged on GitHub into the wrong branch, that should not be in the pull request. I will redo the Pull request on a new branch. Sorry.

**Comment by nanoandrew4** [comment]
I force pushed to rewrite a commit message, that may have messed things up... Didn't expect outside contribution, so didn't bother setting up proper git flow style branches and protections, I'll do that so I can avoid causing issues for other PRs like yours :)

**Pull request closed without merging**